GCSViews/ConfigurationView/ConfigRawParams.cs

```diff
@@ -325,6 +329,18 @@ private void BUT_writePIDS_Click(object sender, EventArgs e)
                 CustomMessageBox.Show("Not all parameters successfully saved.", "Saved");
             else
                 CustomMessageBox.Show("Parameters successfully saved.", "Saved");
+            //Check if reboot is required
+            if (reboot)
+            {
+               CustomMessageBox.Show("Reboot is required for some parameters to take effect.", "Reboot Required");
+            }
+
+            if (MainV2.comPort.MAV.param.TotalReceived != MainV2.comPort.MAV.param.TotalReported)
+            {
+                CustomMessageBox.Show("The number of available parameters changed. A full param refresh will be done to show all params.", "Params");
+                //Click on refresh button
+                BUT_rerequestparams_Click(BUT_rerequestparams, null);

```

Comment:
Refreshing if armed might will flood the link with params, if you have a slow link this will be very painful. You could either not refresh if armed, or if armed pop up a "would you like to refresh?" message to get the user to decide.